src/components/Nav.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './books/books.css';
+
+const Nav = () => (
+  <div className="header">
+    <h1 className="logo" to="/">
+      Bookstore CMS
+    </h1>
+    <nav>
+      <Link className="navItem" to="/">Books</Link>
+      <Link className="navItem" to="/Category">Category</Link>
+    </nav>
+
+  </div>
+);
```

Comment:
## Your Navbar
![image](https://user-images.githubusercontent.com/93039359/232180320-e58f1b6c-c182-42a2-9503-4d86504b91cd.png)

## Zeplin's Navbar
![image](https://user-images.githubusercontent.com/93039359/232180959-1e371b64-9498-4f35-b26b-b1fd50d5abe8.png)

- [ ] It would be nice if you compare Zeplin's Navbar with your navbar and see that your nav links are not at the proper position as it should be according to the Zeplin's design 😊
- [ ] It would be nice if you add the user logo which is at the end of Zeplin's design 👍 kindly add in your design too to make your project follow Zeplin's design 😊